ietf/utils/markdown.py

```diff
@@ -27,23 +37,14 @@ def markdown(text):
                 python_markdown.markdown(
                     text,
                     extensions=[
+                        # TODO: discuss which extensions we want to enable, see
```

Comment:
We did discuss this at the time this went in. I'm ok refreshing the discussion, but what's here isn't whimsical.

Reply:
Want me to remove the comment?